MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -215,30 +240,85 @@ - (BOOL)application:(UIApplication *)app
   return [[MSAppDelegateForwarder sharedInstance] application:app openURL:url options:options returnedValue:result];
 }
 
+- (void)custom_application:(UIApplication *)app didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
```

Comment:
There is one detail that changes is the parameters we have to forward, it can probably be improved but I couldn't find a quick and clean way to do it. I am opened to suggestions.